packages/checkout-ui-extensions/src/components/SkeletonTextBlock/SkeletonTextBlock.ts

```diff
@@ -14,6 +14,10 @@ export interface SkeletonTextBlockProps extends Pick<TextBlockProps, 'size'> {
    * @defaultValue 1
    */
   lines?: number;
+  /**
+   * Size of the text
```

Comment:
```suggestion
   * Size of the skeleton
```

Reply:
Added the changes on checkout-web @jamesvidler